res/storage.py

```diff
@@ -60,67 +59,67 @@ def get_s3_inventory(oId, profile, boto3_config, selected_regions):
 
             bucket_name = bucket['Name']
 
-            # Check if a website if configured; if yes, it could lead to a DLP issue
-            try:
-                has_website = 'unknown'
-                has_website = s3.get_bucket_website(Bucket = bucket_name)
-                del has_website['ResponseMetadata']
-            except ClientError as ce:
-                if 'NoSuchWebsiteConfiguration' in ce.args[0]:
-                    has_website = 'no'
-            bucket['website'] = has_website
+            ## Check if a website if configured; if yes, it could lead to a DLP issue
+            #try:
+            #    has_website = 'unknown'
+            #    has_website = s3.get_bucket_website(Bucket = bucket_name)
+            #    del has_website['ResponseMetadata']
+            #except ClientError as ce:
+            #    if 'NoSuchWebsiteConfiguration' in ce.args[0]:
+            #        has_website = 'no'
+            #bucket['Website'] = has_website
```

Comment:
I believe these sort of changes won't be accepted upstream.
You probably need to introduce a parameter to allow bypassing this block instead of commenting it out.

Reply:
yes it is the goal but I did not proposed this change upstream yet waiting to see how the others are accepted (or not) so I did not invested the time to make the changes generic for now